### PR TITLE
COL-246 Surface points configuration to students who don't share scores

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-246

### CODE REVIEW

File: public/app/engagementindex/leaderboard/leaderboard.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <div data-ng-if="!me.is_admin && me.share_points !== null" data-ng-include="'/app/engagementindex/leaderboard/userinfo.html'"></div>
 
 <!-- LIST -->
-<div data-ng-if="me.is_admin || me.share_points" data-ng-include="'/app/engagementindex/leaderboard/list.html'"></div>
+<div data-ng-include="'/app/engagementindex/leaderboard/list.html'"></div>
```
Comment (nicolaasmatthijs) [inline public/app/engagementindex/leaderboard/leaderboard.html]: This causes the `Points configuration` button to also be present on the `opt in/out` screen students see when to go the Engagement Index for the first time. The `Points configuration` button should not be visible there.

Comment (nicolaasmatthijs) [inline public/app/engagementindex/leaderboard/leaderboard.html]: screen shot 2016-05-19 at 3 31 24 pm

**Comment by nicolaasmatthijs** [comment]
Finished review. Back to @paulkerschen

**Comment by paulkerschen** [comment]
Follow-up pushed, back to @nicolaasmatthijs

**Comment by nicolaasmatthijs** [comment]
Looks good. Merged.

**Pull request merged**